dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -28,8 +28,11 @@ def update
     existing_school_info&.assign_attributes new_school_info_params
     if existing_school_info.nil? || existing_school_info.changed?
       submitted_school_info = SchoolInfo.where(new_school_info_params).
-        first_or_create(validation_type: SchoolInfo::VALIDATION_NONE)
-      current_user.update! school_info: submitted_school_info
+        first_or_create(validation_type: SchoolInfo::VALIDATION_COMPLETE)
```

Comment:
This is where the `validation_type` column will be problematic. There's a part of me that says we should just set that column as `VALIDATION_NONE`.

Sort of on that note, does this do completeness checking or does this just set that column's value?

Reply:
Good point! Initially, I thought about this same issue you raised.  The Validation_completed does a completeness check, which I confirmed with the tests.  So it is working as intended.  Are there any other concerns we should consider?

I vaguely remember the discussion we had on the possibility of removing validation type (not sure if that was the context).

Reply:
Yeah I believe we talked a bit about getting rid of that column, in part for this reason (and afaict it doesn't serve a purpose).